app/pages/Mint/MintBTC.tsx

```diff
@@ -134,7 +139,12 @@ export function MintBTC() {
 			if (response && response.status === "success") {
 				setTxId(response.result.txid);
 				setShowConfirmationModal(true);
-				if (response.result.txid) await putNBTCTX(response.result.txid, network);
+				if (response.result.txid)
+					await makeReq(mintTxFetcher, {
+						method: "putNBTCTx",
+						params: [network, response.result.txid],
+					});
+				fetchMintTxs();
```

Comment:
why we are quering this? No need to do a second query, just add it based on `putNBTCTx` response and reduce reloads.